ethereum_history_api/circuits/lib/src/rlp/types_test.nr

```diff
@@ -114,4 +260,32 @@ mod rlp_fragment {
             fragment.assert_eq_address("Field", address, alter_array(address))
         }
     }
+
+    mod assert_eq_bytes32 {
+        use crate::rlp::{decode::STRING, types::RlpFragment};
+        use crate::misc::types::Bytes32;
+        use crate::misc::arrays::alter_array;
+
+        global bytes = [
+            0x88, 0xe9, 0x6d, 0x45, 0x37, 0xbe, 0xa4, 0xd9, 0xc0, 0x5d, 0x12, 0x54, 0x99, 0x07, 0xb3, 0x25, 0x61, 0xd3, 0xbf, 0x31, 0xf4, 0x5a, 0xae, 0x73, 0x4c, 0xdc, 0x11, 0x9f, 0x13, 0x40, 0x6c, 0xb6
```

Comment:
Where does this come from? Is this some block hash?

Reply:
Yes. Just like with address in tests above, I used some real data.

Reply:
Can we import it from fixtures?